xpk.py

```diff
@@ -2506,6 +2506,51 @@ def create_cluster_if_necessary(args) -> int:
     return run_gke_cluster_create_command(args)
 
 
+def get_nodepool_zone(args, nodepool_name) -> tuple[int, str]:
+  """Return zone in which nodepool exists in the cluster.
+
+  Args:
+    args: user provided arguments for running the command.
+    nodepool_name: name of nodepool.
+
+  Returns:
+    0 if successful, 1 otherwise and the zone of nodepool.
```

Comment:
nit that the function also returns a string

Reply:
I have mentioned `zone of nodepool`. I will add this as a new line for readability.